### PR TITLE
Honeypot

### PR DESCRIPTION
**Pull request by mikeyrayvon**
idk scott wanted to do something with this image of a honeypot to correspond with the mention of 'honeypot' in the text. so this is what I came up with.

you need to have a `<span id="honeypot">` in one of the Home text fields. You can use this:

```
When we hit a <a id="honeypot">honeypot</a> of something extra special, we reserve it all for our preferred clients. After a long time of working this way our most loved cultivators have evolved to producing solely for us. Maison Gas is how we offer the certain level of quality we look for exclusively to you.
```

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -37,6 +37,45 @@ class Site {
...
+    const $honeypot = $('#honeypot');
+
+    $honeypot.on('click', this.handleHoneypot.bind(this));
+  }
+
+  handleHoneypot() {
+    let honeypotCount = 0;
+    const honeypotInterval = setInterval(() => {
+      // oh well hello
+      const $honeypotImage = $('<img src="' + WP.themeUrl + '/dist/img/honeypot.jpg" class="honeypot" />');
```
Comment (varffs) [inline src/js/main.js]: I feel like maybe this image should/could come from a site option?

File: src/js/main.js [hunk 2]
```diff
@@ -37,6 +37,45 @@ class Site {
...
+
+      // oooh that's sweet
+      $('body').append($honeypotImage);
```
Comment (varffs) [inline src/js/main.js]: I think these appended images need to be removed as if you click it a few times you can end up with hundreds of `opacity: 0` images in your DOM lol.

File: src/styl/responsive/mobile.styl [hunk 3]
```diff
@@ -27,6 +27,24 @@ p
   p
     hyphens: none
 
+#honeypot
```
Comment (varffs) [inline src/styl/responsive/mobile.styl]: I wonder if the trigger should be a class so they could potentially have the text and effect in multiple locations?

**Comment by mikeyrayvon** [comment]
all great ideas. i will implement

**Comment by mikeyrayvon** [comment]
ok ya'll i think this is ready to merge

**Pull request merged**